pages/common/acme.sh.md

```diff
@@ -23,13 +24,9 @@
 
 `acme.sh --issue --apache --domain {{example.com}}`
 
-- Issue a wildcard (\*) certificate using a manual DNS mode:
+- Issue a wildcard (\*) certificate using an automatic DNS API mode:
 
-`acme.sh --issue --dns --domain {{example.com}}`
-
-- Issue a certificate using an automatic DNS API mode:
-
-`acme.sh --issue --dns {{dns_cf}} --domain {{example.com}}`
+`acme.sh --issue --dns {{dns_cf}} --domain {{*.example.com}}`
 
 - Install certificate files into the specified locations (useful for automatic certificate renewal):
```

Comment:
Btw, shouldn't `the` in `the specified locations` be omitted?

Reply:
@peterbabic no, I don't see any reason to.  The text can be _slightly_ informal—articles and contractions are usually the most we do.